src/Suave.Tests/Types.fs

```diff
@@ -57,6 +57,11 @@ let httpBinding (_ : SuaveConfig) =
       let binding = HttpBinding.create HTTP IPAddress.Loopback 80us
       let actual = binding.uri "" "" |> sprintf "%O"
       Expect.equal actual "http://127.0.0.1/" "uri"
+
```

Comment:
Huh, turns out in this case the unescaping in HttpBinding.uri is unnecessary as well. I encountered a test which expected unescaped results but seems it was when I handled the incoming URI differently. Would the test suite be OK like this if I remove the unescaping from the implementation?